.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+## Proposed Changes
+
+- Change
+- Update
+- Fix
+
+## Relates To
+
+- Issue #0
+
+## Notify
+
+- @octocat
+- @beetboxvm/octoteam
```

Comment:
As I'm not a member of the beetboxvm organisation, I can't see your / their teams. If I could, I'd tell you that is probably a non-existent team. 

Octocat is also an account you probably wouldn't notify. 

These are what are called examples.